crates/matrix-sdk-base/src/store/integration_tests.rs

```diff
@@ -277,18 +277,18 @@ macro_rules! statestore_integration_tests {
 
                     assert!(store.get_sync_token().await?.is_some());
                     assert!(store.get_presence_event(user_id).await?.is_some());
-                    assert_eq!(store.get_room_infos().await?.len(), 1);
-                    assert_eq!(store.get_stripped_room_infos().await?.len(), 1);
+                    assert_eq!(store.get_room_infos().await?.len(), 1,  "Expected to find 1 room info ");
+                    assert_eq!(store.get_stripped_room_infos().await?.len(), 1,  "Expected to find 1 stripped room info");
```

Comment:
Still extra spaces here.

```suggestion
                    assert_eq!(store.get_room_infos().await?.len(), 1, "Expected to find 1 room info ");
                    assert_eq!(store.get_stripped_room_infos().await?.len(), 1, "Expected to find 1 stripped room info");
```